libethereum/Client.cpp

```diff
@@ -659,7 +658,7 @@ void Client::doWork(bool _doWait)
 	bool isSealed = false;
 	DEV_READ_GUARDED(x_working)
 		isSealed = m_working.isSealed();
-	if (!isSealed && !isSyncing() && !m_remoteWorking && m_syncTransactionQueue.compare_exchange_strong(t, false))
```

Comment:
This will make processing of pending transactions to start closer to sync finish.
(`isSyncing()` returns `false` when there are no more blocks to download, but there still may be many blocks to import in `BlockQueue`)